cypress/e2e/assets_spec/asset_homepage.cy.ts

```diff
@@ -16,7 +16,7 @@ describe("Asset Tab", () => {
   const assetFilters = new AssetFilters();
   const assetPage = new AssetPage();
   const loginPage = new LoginPage();
-  const assetName = "Dummy Camera 10";
```

Comment:
@kshitijv256 why are we changing this

Reply:
There was no asset with the given name so changed to what was available